pyproject.toml

```diff
@@ -1,3 +1,32 @@
+[tool.poetry]
+name = "pandas-stubs"
+version = "0.0.1"
```

Comment:
I'd like to use a version numbering convention of pandas_version.yymmdd for the stubs.  So if these were done today, the version would be 1.4.2.220607 .  Then we mark the releases of the stubs by their release date, and also know which version of pandas the stubs correspond to.

Reply:
I did not work with the poetry version yet, because we did not have a convention yet and my future ideia is to automate the version, tag, and publish system, so we do not need to worry with the static version in pyproject.toml file.

Reply:
> Oh another important point to work is to upgrade the current stub version to work in 1.4.2 version, I guess you already saw it, but it is compatible just with API 1.2.x.

What makes you believe it is not compatible with 1.4?  Are we missing something?  

The stubs that are here came from the Microsoft repo  `https://github.com/microsoft/python-type-stubs` (and they just retired the pandas stubs there).  I've used them successfully in projects using pandas 1.4.2, so I'd like to understand your comment about compatibility with API versions.